hexrd/unitcell.py

```diff
@@ -1480,9 +1479,10 @@ def Required_C(self, C):
 
     def MakeStiffnessMatrix(self, inp_Cvals):
         if(len(inp_Cvals) != len(_StiffnessDict[self._laueGroup][0])):
-            raise IOError('number of constants entered is not correct.\
-            need a total of ', len(_StiffnessDict[self._laueGroup][0]),
-                          'independent constants')
+            x = len(_StiffnessDict[self._laueGroup][0])
+            msg = f"number of constants entered is not correct."
+            "need a total of {x} independent constants"
```

Comment:
I think multi-line strings need to be in parentheses. And each f-string needs to start with "f", like:

```python
            msg = (f"number of constants entered is not correct. "
                   f"Need a total of {x} independent constants")
```